src/components/Builder/Social/Social.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+
+import { ReactComponent as Website } from "../../../assets/website.svg";
+import { ReactComponent as Facebook } from "../../../assets/facebook.svg";
+import { ReactComponent as Instagram } from "../../../assets/instagram.svg";
+import { ReactComponent as Twitter } from "../../../assets/twitter.svg";
+
+const Social = ({ parent }) => {
+  return (
+    <div className="social">
+      {!!parent.website && (
+        <Link className="social__item" to={parent.website}>
+          <Website />
+        </Link>
+      )}
+      {!!parent.facebook && (
+        <Link className="social__item" to={parent.facebook}>
+          <Facebook />
+        </Link>
+      )}
+      {!!parent.instagram && (
+        <Link className="social__item" to={parent.instagram}>
+          <Instagram />
+        </Link>
+      )}
+      {!!parent.twitter && (
+        <Link className="social__item" to={parent.twitter}>
+          <Twitter />
+        </Link>
+      )}
+    </div>
+  );
+};
+export default withRouter(Social);
```

Comment:
This withRouter functionality doenst seem necessary, we should pass the data a props